README.md

```diff
@@ -108,3 +108,5 @@ To enable tracing, configure the Jaeger Tracer using [environment variables](htt
 
 To run Jaeger Tracing, you can also use the provided example in [`deployment-tracing-jaeger.yaml`](./java/kafka/deployment-tracing-jaeger.yaml).
 To run Opentelemetry Tracing, you can also use the provided example in [`deployment-tracing-opentelemetry.yaml`](./java/kafka/deployment-tracing-opentelemetry.yaml).
+
+*NOTE* : Jaeger Tracink eg is only supported up to Kafka version 2.8.0. For Kafka 2.8 +, use OpenTelemetry.
```

Comment:
I thinks this should be removed, see @scholzj comment here as well https://github.com/strimzi/client-examples/pull/86#discussion_r1000747589

Reply:
@ppatierno @scholzj @kyguy We know that Jaeger still works with consumer and producer but not with streams in version 3.3. Should I replace the comment in the README with something else or just remove completely?

Reply:
First of all, there is no versioning for the client examples. Just the source codes. So the reference to 2.8 makes no sense as there is no way to use the 2.8 version. Also, OpenTelemetry and OpenTracing do the same, but are not necessarily one to one replacement.

I don't think we should block the progress => so I would suggest to:
* Remove OpenTracing support from the Streams example
* Keep it in the Producer / Consumer examples where it IMHO still works fine
* Add a note to the README in this sense (=> Jaeger / OpenTracing tracing is supported only in consumers / producers because OPenTracing support for KAfka Streams API is not compatible with latest Kafka versions)

Reply:
Thanks @scholzj. Changes made, tested and pushed.